src/entities/token.ts

```diff
@@ -10,6 +10,47 @@ export class Token extends Currency {
   public readonly chainId: ChainId
   public readonly address: string
 
+  public static readonly WETH: { [key: number]: Token } = {
+    [ChainId.MAINNET]: new Token(
+      ChainId.MAINNET,
+      '0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2',
+      18,
+      'WETH',
+      'Wrapped Ether'
+    ),
+    [ChainId.RINKEBY]: new Token(
+      ChainId.RINKEBY,
+      '0xc778417E063141139Fce010982780140Aa0cD5Ab',
+      18,
+      'WETH',
+      'Wrapped Ether'
+    ),
+    [ChainId.ARBITRUM_TESTNET_V3]: new Token(
+      ChainId.ARBITRUM_TESTNET_V3,
+      '0xf8456e5e6A225C2C1D74D8C9a4cB2B1d5dc1153b',
+      18,
+      'WETH',
+      'Wrapped Ether'
+    ),
+    [ChainId.SOKOL]: new Token(ChainId.SOKOL, '0xfDc50eF6b67F65Dddc36e56729a9D07BAe1A1f68', 18, 'WETH', 'Wrapped Ether')
+  }
+
+  public static readonly WSPOA: { [key: number]: Token } = {
+    [ChainId.SOKOL]: new Token(ChainId.SOKOL, '0xc655c6D80ac92d75fBF4F40e95280aEb855B1E87', 18, 'WSPOA', 'Wrapped SPOA')
+  }
+
+  public static readonly DXD: { [key: number]: Token } = {
+    [ChainId.MAINNET]: new Token(ChainId.MAINNET, '0xa1d65E8fB6e87b60FECCBc582F7f97804B725521', 18, 'DXD', 'DXdao'),
+    [ChainId.RINKEBY]: new Token(ChainId.RINKEBY, '0x554898A0BF98aB0C03ff86C7DccBE29269cc4d29', 18, 'DXD', 'DXdao')
+  }
+
+  private static readonly NATIVE_CURRENCY_WRAPPER: { [chainId in ChainId]: Token } = {
```

Comment:
Are we missing xdai on this list?

Reply:
Oh, I will add it as soon as I actually deploy the contracts to xDAI and make the app work on there too (so most likely today).